llama_index/multi_modal_llms/fuyu.py

```diff
@@ -0,0 +1,167 @@
+from typing import Any, Callable, Dict, Optional
+
+from llama_index.bridge.pydantic import Field, PrivateAttr
+from llama_index.callbacks import CallbackManager
+from llama_index.constants import DEFAULT_CONTEXT_WINDOW, DEFAULT_NUM_OUTPUTS
+from llama_index.llms.generic_utils import (
+    messages_to_prompt as generic_messages_to_prompt,
+)
+from llama_index.multi_modal_llms import (
+    MultiModalCompletionResponse,
+    MultiModalCompletionResponseGen,
+    MultiModalLLMMetadata,
+)
+from llama_index.schema import ImageDocument
+
+
+class Fuyu(MultiModalLLMMetadata):
```

Comment:
I wonder if we can generalize this class to just multimodal models from hugginface. And then just a have a type alias, or subclass for `Fuyu`.